### PR TITLE
204 add more words

### PR DESCRIPTION
**Pull request by omen273**

### CODE REVIEW

File: app/src/main/res/raw/data.json [hunk 1]
```diff
@@ -12691,5 +12691,973 @@
...
+            "word": "plus",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "upper-intermediate(B2)"
+        }},
+        {"RU": {
+            "word": "плюс",
+            "topics": ["математика и измерения", "наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:13553]:
```suggestion
    ],
	[
```

Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:13553]: already have

Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:13553]: also why B2? I see B1 in dictionary

File: app/src/main/res/raw/data.json [hunk 2]
```diff
@@ -12691,5 +12691,973 @@
...
+            "word": "liter",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "upper-intermediate(B2)"
+        }},
+        {"RU": {
+            "word": "литр",
+            "topics": ["математика и измерения", "наука и техника"]
+        }}
+    ],
+	[
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:13454]:
```suggestion
    ],
	[
```

Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:13454]: also duplicate and it should be B1

File: app/src/main/res/raw/data.json [hunk 3]
```diff
@@ -12691,5 +12691,973 @@
...
+            "level": "upper-intermediate(B2)"
+        }},
+        {"RU": {
+            "word": "материя",
+            "topics": ["физика и химия", "наука и техника"]
+        }}
+    ],
+	[
+        {"EN":{
+            "word": "melt",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:13104]: I think it might be unified with cooking

File: app/src/main/res/raw/data.json [hunk 4]
```diff
@@ -12691,5 +12691,973 @@
...
-    ]
+    ],
+	[
+        {"EN":{
+            "word": "activate",
+            "topics": ["engineering", "physics and chemistry", "science and technology"],
+            "level": "upper-intermediate(B2)"
+        }},
+        {"RU": {
+            "word": "включать",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:12702]: активировать? в примере есть ген например, про него сложно сказать "включать"

File: app/src/main/res/raw/data.json [hunk 5]
```diff
@@ -12691,5 +12691,973 @@
...
+        }}
+    ],
+	[
+        {"EN":{
+            "word": "crack",
+            "topics": ["physics and chemistry", "science and technology"],
+            "level": "upper-intermediate(B2)"
+        }},
+        {"RU": {
+            "word": "ломать",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:13021]: трескаться лучше кмк

File: app/src/main/res/raw/data.json [hunk 6]
```diff
@@ -12691,5 +12691,973 @@
...
+        }}
+    ],
+	[
+        {"EN":{
+            "word": "count",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "upper-intermediate(B2)"
+        }},
+        {"RU": {
+            "word": "число",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:13351]: подсчет?

File: app/src/main/res/raw/data.json [hunk 7]
```diff
@@ -12691,5 +12691,973 @@
...
+        }}
+    ],
+	[
+        {"EN":{
+            "word": "unit",
+            "topics": ["maths and measurement", "science and technology"],
+            "level": "upper-intermediate(B2)"
+        }},
+        {"RU": {
+            "word": "цифра",
```
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:13637]:
```suggestion
            "word": "единица",
```

File: app/src/main/res/raw/data.json (hunk 6 above)
Comment (omen273) [reply app/src/main/res/raw/data.json:13351]: Decided to stay as it is

File: app/src/main/res/raw/data.json (hunk 7 above)
Comment (omen273) [reply app/src/main/res/raw/data.json:13637]: Decided to stay as it is

File: app/src/main/res/raw/data.json (hunk 3 above)
Comment (omen273) [reply app/src/main/res/raw/data.json:13104]: Decided to stay as it is

File: app/src/main/res/raw/data.json (hunk 5 above)
Comment (greenEkatherine) [inline app/src/main/res/raw/data.json:13021]:
```suggestion
            "word": "трескаться",
```

**Review by greenEkatherine — APPROVED** [review]

**Pull request merged**